dev/com.ibm.ws.security.social/resources/com/ibm/ws/security/social/resources/SocialMessages.nlsprops

```diff
@@ -655,3 +655,8 @@ BACKCHANNEL_REQUEST_NOT_SUPPORTED_CONFIG.useraction=Use the logout endpoint that
 CLIENT_SECRET_MISSING_BUT_REQUIRED_BY_TOKEN_AUTH_METHOD=CWWKS2351E: The {0} OpenID Connect client uses the {1} token endpoint authentication method. This authentication method requires a client secret, but a client secret is not configured.
 CLIENT_SECRET_MISSING_BUT_REQUIRED_BY_TOKEN_AUTH_METHOD.explanation=The token endpoint authentication method that is specified in the message requires a client secret to authenticate with the token endpoint of the OpenID Connect provider.
 CLIENT_SECRET_MISSING_BUT_REQUIRED_BY_TOKEN_AUTH_METHOD.useraction=Configure a client secret for the OpenID Connect client, or switch the token endpoint authentication method of the client to one that does not require a client secret.
+
+# do not translate hostname, wasReqURLRedirectDomainNames, and webAppSecurity
+MALFORMED_URL_IN_ORIGIN_REQUEST_URL_COOKIE=CWWKS2352E: A request to [{0}] is not valid. A required cookie with a name that begins with [{1}] has an incorrect value. The host name [{2}] that is used in the cookie does not match the name that is registered at the provider.
+MALFORMED_URL_IN_ORIGIN_REQUEST_URL_COOKIE.explanation=A request was received that includes a malformed cookie.
+MALFORMED_URL_IN_ORIGIN_REQUEST_URL_COOKIE.useraction=Verify the OpenID Connect provider and client configurations. The malformed cookie can be caused by cookie modification at the user agent with a host name that differs from the host name of the redirect that is registered with the provider. If the host name is expected, then add it to the wasReqURLRedirectDomainNames attribute of the webAppSecurity element in the server configuration.
```

Comment:
The second sentence seems like it could be tightened up. Is it correct to say this?
A user agent can malform the cookie with a hostname that differs from the hostname of the redirect that is registered with the provider.

also, in 3rd sentence, change
host name
to
hostname